pkg/controller/database/rdsinstance.go

```diff
@@ -209,7 +170,6 @@ func (e *external) Create(ctx context.Context, mg resource.Managed) (managed.Ext
 		return managed.ExternalCreation{}, errors.New(errNotRDSInstance)
 	}
 
-	cr.SetConditions(xpv1.Creating())
```

Comment:
These are now set automatically 👍 https://github.com/crossplane/crossplane-runtime/pull/242

Reply:
Reverted the switch statement and removed `creating` and `deleting` condition setting lines.